rootfs_skeleton/var/web/mosquitto_mqtt_broker/d

```diff
@@ -2,25 +2,27 @@ ACTIVATE_SETTINGS=Einstellungen aktivieren
 ADMINISTRATION=Administration
 APPLICATION=Anwendung
 AUTHENTICATION=Authentifizierung
-BIND_ADDRESS=Bind Address
+BIND_ADDRESS=Bind address
 CONFIGURATION=Konfiguration
-DNS=DNS-Server-Adresse
+DNS=DNS-Server
 ENGLISH=Englisch
 GATEWAY=Default-Gateway
 GERMAN=Deutsch
 HTML_TITLE=INSYS Mosquitto MQTT-Broker (Demonstrator)
 IP_ADDRESS=IP-Adresse
 IP_NETWORK=IP-Netzwerk
+LOGGING=Log-Ansicht
 OVERVIEW=&Uuml;bersicht
 PORT=Port
 RESTART=Neustart
 RESTARTING_DEVICE=Der Container wird neu gestartet.<br>Dies wird ein paar Sekunden dauern.<br>
-RESTART_APP=Applikation jetzt neu starten
-RESTART_CONTAINER=Container jetzt neu starten
+RESTART_APP=Anwendung neu starten
```

Comment:
Nur als Info: Das "jetzt" haben wir damals mit einem eigenen Commit in der Firmware eingeführt, um einen Unterscheidung zu dem Geklickere auf das Zahnrad zu haben. Wenn man auf OK klickt, führt man nur "Speichern" aus, mit dem Klick auf das Zahnrad wird das gespeicherte Profil erst aktiv. Mit dem Button "jetzt neu starten" hat man eine dritte Option - "jetzt sofort ausführen".
Ist aber wohl für diese Applikation ned ganz so wichtig.